content/blog/2018/07/2018-07-25-contributor-summit.adoc

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "Join us at the Jenkins Contributor Summit San Francisco, Monday 17 September 2018"
+tags:
+- community
+- events
+- jenkins-world
+author: tracymiranda
+---
+
+The Jenkins Contributor summit is where the current and future contributors of the Jenkins project get together.
+This summit will be on Monday, September 17th 2018 in San Francisco, just before link:https://www.cloudbees.com/devops-world/san-francisco[DevOps World | Jenkins World].
+The summit brings together community members to learn, meet and help shape the future of Jenkins. 
+In the Jenkins commmunity we value all types and sizes of contributions and love to welcome new participants.
+Register link:https://www.eventbrite.com/e/jenkins-contributor-summit-san-francisco-tickets-48285340754[here].
+
+== Topics
+There are plenty of exciting developments happening in the Jenkins community. 
+The summit will feature a 'State of the Project' update including updates from the Jenkins officers. 
+We will also have updates on the 'Big 5' projects in active development:
+* link:https://jenkins.io/blog/2018/04/06/jenkins-essentials[Jenkins Essentials]
+* link:https://jenkins-x.io[Jenkins X]
+* link:https://www.praqma.com/stories/jenkins-configuration-as-code[Configuration as Code]
+* Jenkins Pipeline
+* Cloud Native Jenkins Architecture
+
+Plus we will feature a Google Summer of Code update, and more!
+
+== Agenda
+The agenda is shaping up well and here is the outline so far.
+
+*  9:00am Kickoff & Welcome with coffee/pastries
+* 10:00am Project Updates
+* 12:00pm Lunch
+*  1.00pm BoF/Unconference
+*  3.00pm Break
+*  3.30pm Ignite Talks
+*  5.00pm Wrap-up
+*  6.00pm Contributor Dinner
+
+The BoF (birds-of-a-feather) session will be an opportunity for in depth discussions, hacking or learning more about any of the big 5. 
+Bring your laptop, come prepared with questions and ideas, and be ready for some hacking too if you want.
+Join in, hear the latest and get involved in any project during the BoF sessions.
+If you want to share anything there will be an opportunity to do a 5-min ignite talk at the end.
+Attending is free, and no DevOps World | JenkinsWorld ticket is needed, but link:https://www.eventbrite.com/e/jenkins-contributor-summit-san-francisco-tickets-48285340754[RSVP] if you are going to attend to help us plan.

```

Comment:
"JenkinsWorld" no line break

Reply:
While yes we can and the agenda is meant to evolve, for this post the purpose is to get the date out there and let folks start registering. In fact we have a different agenda on eventbrite page so separately let's get this out and use it as a reference point when we are adding/removing things to it.